src/QMCHamiltonians/ECPotentialBuilder.cpp

```diff
@@ -73,6 +74,7 @@ bool ECPotentialBuilder::put(xmlNodePtr cur)
   pAttrib.add(pbc, "pbc", {"yes", "no"});
   pAttrib.add(forces, "forces", {"no", "yes"});
   pAttrib.add(physicalSO, "physicalSO", {"yes", "no"});
+  pAttrib.add(fast_so, "fastSO", {"yes", "no"});
```

Comment:
"fastSO" is hard to understand what is fast. Could you find a more understandable name?

Reply:
Switched to spin_integrator. Can choose between "exact" and "simpson" which will use the old simpsos rule to integrate